onnxruntime/python/tools/transformers/bert_perf_test.py

```diff
@@ -275,6 +292,17 @@ def run_one_test(model_setting, test_setting, perf_results, all_inputs, intra_op
         "Average latency = {} ms, Throughput = {} QPS".format(format(average_latency, ".2f"), format(throughput, ".2f"))
     )
 
+    if model_setting.write_tuning_results:
+        output_path = model_setting.model_path + f"tuning_results_{os.getpid()}.json"
+        if os.path.exists(output_path):
+            print(output_path, "exists, skip tuning results writing.")
+            return
+
+        trs = session.get_tuning_results()
+        print("Tuning results is saved to", output_path)
```

Comment:
This line could be moved after saving is done. Otherwise, change the wording to "is saving to".